src/service.rs

```diff
@@ -679,7 +650,11 @@ fn build_query_options(
         Order::Desc => after_cursor.map_or_else(
             || {
                 (
-                    [prefix.clone(), vec![0xff; MAX_PREFIX_SEARCH_SIZE]].concat(),
+                    [
+                        prefix.clone(),
+                        vec![0xff; MAX_PREFIX_SEARCH_SIZE - args_len],
```

Comment:
If there is a `args = vec![0xff ; MAX_PREFIX_SEARCH_SIZE - args_len]` this `after_cursor` will missing this key.
Suggestion
```
after_cursor_args = (BigInt(script.args.raw_data()) + 1).to_hex()
```